app-libs/stf/poc/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(not(feature = "std"))]
+extern crate alloc;
+
+use log::info;
+use match_test::{self, from_json_string, to_json_string, Person};
+
```

Comment:
If you want you can re-export `Person` here, then you don't need to add `match_test` in other crates as a dependency. Re-exporting is done with:


```rust
pub use match_test::Person;
```